akka-http-core/src/main/scala/akka/http/impl/util/package.scala

```diff
@@ -109,10 +98,10 @@ package object util {
     }
   }
 
-  private[http] def errorLogger(log: LoggingAdapter, msg: String): PushStage[ByteString, ByteString] =
-    new PushStage[ByteString, ByteString] {
-      override def onPush(element: ByteString, ctx: Context[ByteString]): SyncDirective = ctx.push(element)
-      override def onUpstreamFailure(cause: Throwable, ctx: Context[ByteString]): TerminationDirective = {
+  private[http] def errorLogger[T](log: LoggingAdapter, msg: String): PushStage[T, T] =
+    new PushStage[T, T] {
+      override def onPush(element: T, ctx: Context[T]): SyncDirective = ctx.push(element)
+      override def onUpstreamFailure(cause: Throwable, ctx: Context[T]): TerminationDirective = {
```

Comment:
@jrudolph This should be replaced with `.log()` - but there's a separate ticket for it so no worries about it in this PR: https://github.com/akka/akka/issues/17184